v2/concordium/types.proto

```diff
@@ -2186,6 +2186,10 @@ message PoolCurrentPaydayInfo {
   Amount delegated_capital = 7;
   // The commission rates that apply for the current reward period.
   CommissionRates commission_rates = 8;
+  // A flag indicating whether the pool owner is primed for suspension.
+  bool is_primed_for_suspension = 9;
+  // The number of missed rounds in the current reward period.
+  uint64 missed_rounds = 10;
```

Comment:
I suggest making these fields optional, and only populate them when the protocol version supports it.
```suggestion
  // A flag indicating whether the pool owner is primed for suspension.
  // Absent if the protocol version does not support validator suspension.
  optional bool is_primed_for_suspension = 9;
  // The number of missed rounds in the current reward period.
  // Absent if the protocol version does not support validator suspension.
  optional uint64 missed_rounds = 10;
```